integ-test/src/test/java/org/opensearch/sql/sql/HighlightFunctionIT.java

```diff
@@ -35,54 +35,51 @@ public void accepts_unquoted_test() {
     String query = "SELECT Tags, highlight(Tags) FROM %s WHERE match(Tags, 'yeast') LIMIT 1";
     JSONObject response = executeJdbcRequest(String.format(query, TestsConstants.TEST_INDEX_BEER));
     verifySchema(response, schema("Tags", null, "text"),
-        schema("highlight(Tags)", null, "nested"));
+        schema("highlight(Tags)", null, "keyword"));
```

Comment:
I believe this needs to stay as `nested` not `keyword` because this is a change to a published interface and because `keyword` is a string type, but highlight is a tuple.

Reply:
I believe the type is correct. 

`'{"query":"select conv(11, 16, 10);"}'`
```
{
  "schema": [
    {
      "name": "conv(11, 16, 10)",
      "type": "keyword"
    }
  ],
  "datarows": [
    [
      "17"
    ]
  ],
  "total": 1,
  "size": 1,
  "status": 200
}
```

`'{"query":"SELECT highlight('Tags') FROM beer.stackexchange WHERE match(Tags, 'yeast') LIMIT 1;"}'`
```
{
  "schema": [
    {
      "name": "highlight(Tags)",
      "type": "keyword"
    }
  ],
  "datarows": [
    [
      [
        "health <em>yeast</em>"
      ]
    ]
  ],
  "total": 1,
  "size": 1,
  "status": 200
}
```

Here is the output from the previously released version:

`{"query":"SELECT highlight('Tags') FROM beer.stackexchange WHERE match(Tags, 'yeast') LIMIT 1;"}'`
```
{
  "schema": [
    {
      "name": "highlight(Tags)",
      "type": "nested"
    }
  ],
  "datarows": [
    [
      [
        "health <em>yeast</em>"
      ]
    ]
  ],
  "total": 1,
  "size": 1,
  "status": 200
}
```

The previous implementation used the wrong type.